manifests/example-dev/ide-modules.yaml

```diff
@@ -1,97 +1,97 @@
-#name: jupyter-hub
-#path: modules/demo-only/jupyter-hub/
-#parameters:
-#  - name: eks-cluster-admin-role-arn
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksClusterAdminRoleArn
-#  - name: eks-cluster-name
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksClusterName
-#  - name: eks-oidc-arn
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksOidcArn
-#  - name: secrets-manager-name
-#    value: jh-credentials
-#---
-#name: emrstudio-on-eks
-#path: modules/beta/emrstudio-on-eks/
-#parameters:
-#  - name: vpc-id
-#    valueFrom:
-#      moduleMetadata:
-#        group: optionals
-#        name: networking
-#        key: VpcId
-#  - name: private-subnet-ids
-#    valueFrom:
-#      moduleMetadata:
-#        group: optionals
-#        name: networking
-#        key: PrivateSubnetIds
-#  - name: eks-cluster-admin-role-arn
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksClusterAdminRoleArn
-#  - name: eks-cluster-name
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksClusterName
-#  - name: eks-oidc-arn
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksOidcArn
-#  - name: eks-openid-issuer
-#    valueFrom:
-#      moduleMetadata:
-#        group: core
-#        name: eks
-#        key: EksClusterOpenIdConnectIssuer
-#  - name: artifact-bucket-name
-#    valueFrom:
-#      moduleMetadata:
-#        group: optionals
-#        name: datalake-buckets
-#        key: ArtifactsBucketName
-#  - name: sso-username
-#    value: replace-me
-#  - name: emr-eks-namespace
-#    value: emr-studio
-# ---
-# name: vscode
-# path: modules/demo-only/vscode-on-eks/
-# parameters:
-#   - name: eks-cluster-admin-role-arn
-#     valueFrom:
-#       moduleMetadata:
-#         group: core
-#         name: eks
-#         key: EksClusterAdminRoleArn
-#   - name: eks-cluster-name
-#     valueFrom:
-#       moduleMetadata:
-#         group: core
-#         name: eks
-#         key: EksClusterName
-#   - name: eks-oidc-arn
-#     valueFrom:
-#       moduleMetadata:
-#         group: core
-#         name: eks
-#         key: EksOidcArn
-#   - name: secrets-manager-name
-#     value: vscode-credentials
\ No newline at end of file
+name: jupyter-hub
```

Comment:
are we including these again?

Reply:
I have commented the `ide-modules.yaml` in dep.yaml which would not deploy ide modules instead of commenting the content the manifest files.